src/core/cluster.py

```diff
@@ -404,10 +404,10 @@ def racks(self) -> int:
         return len({broker.rack for broker in self.brokers if broker.rack})
 
     @property
-    def broker_capacities(self) -> dict[str, list[JSON]]:
+    def broker_capacities(self) -> BrokerCapacities:
         """The capacities for all Kafka broker."""
         broker_capacities = []
-        for broker in self.brokers:
+        for broker in sorted(self.brokers, key=lambda broker: broker.unit_id, reverse=True):
```

Comment:
I could have used `operator.itemgetter` instead of a lambda, but I do not mind them. Please let me know if you ever find lambda cumbersome and difficult to read